tests/integration/docusaurus/tutorials/quickstart/v1_pandas_quickstart.py

```diff
@@ -0,0 +1,65 @@
+"""
+This guide demonstrates how to use Great Expectations in a quick-start
+notebook environment with Pandas. It is useful to explore and understand how Great Expectations
```

Comment:
Separated out Pandas vs SQL tests due to how the test script runner requires dependencies to be explicitly stated